pkg/helper/envVar.go

```diff
@@ -1,8 +1,6 @@
 package helper
 
-import (
-	"k8s.io/kubernetes/pkg/api"
-)
+import "k8s.io/kubernetes/pkg/api"
```

Comment:
One reference to how the parentheses-based approach is simpler, especially if you ever plan on importing multiple packages: https://github.com/golang/go/issues/13813